features/storage/blockdevice/BufferedBlockDevice.cpp

```diff
@@ -26,7 +26,8 @@ static inline uint32_t align_down(bd_size_t val, bd_size_t size)
 }
 
 BufferedBlockDevice::BufferedBlockDevice(BlockDevice *bd)
-    : _bd(bd), _bd_program_size(0), _curr_aligned_addr(0), _flushed(true), _cache(0), _init_ref_count(0), _is_initialized(false)
+    : _bd(bd), _bd_program_size(0), _bd_read_size(0), _write_cache_addr(0), _write_cache_valid(false),
```

Comment:
Does this constructor change not need new documentation?

Reply:
It doesn't look like the constructor params changed. These are [member initializers](https://en.cppreference.com/w/cpp/language/initializer_list), which are internal.

Reply:
Curse that lone `:`